### ISSUE TITLE
Specifically address the accessibility benefits of responsive design

### ISSUE THREAD

**Issue by lakeen** [w3c/wai-bad-demo#3]
Under:

Notes, issues
• The content of the demo is layered with accessible, inaccessible, and annotated views; may not be possible to use fully responsive design. 

I think we should highlight the difference between a site that is not coded to be responsive and one that is responsive increases the accessibility.  Therefore only the "after" site should be responsive to highlight the difference.  I imagine that is the intent of the note above but could be re-written to be more specific.

**Comment by bakkenb** [comment]
When I read the Note, the way I interpreted it was because the site would include an inaccessible group of pages (to demonstrate the "Before"), that it would not be very conducive to responsive design so that viewers of the BAD site would see the "Before" pages very well on smaller viewports.

That being said, because of your comment Laura, maybe that is okay that the "Before" pages would not be responsive, therefore driving home the issue to anyone trying to view and use the BAD site on a mobile device vs. a laptop or desktop.
